documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -1,206 +1,59 @@
-	--datadir=/usr/local/share/qemu-lite \
-	--libdir=/usr/local/lib64/qemu-lite \
-	--libexecdir=/usr/local/libexec/qemu-lite
-$ make -j5 && sudo make install
-$ popd
-```
-
-## Install container kernel and rootfs
-
-```
-$ sudo yum-config-manager --add-repo http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/RHEL_7/home:clearlinux:preview:clear-containers-2.1.repo
-$ sudo yum update
-$ sudo yum install linux-container clear-containers-image clear-containers-selinux
-
-```
+For more information on docker installation, see:
+- https://docs.docker.com/engine/installation/linux/rhel/
 
-## Install latest Golang
+## Installation
 
-cc-oci-runtime supports golang 1.7 and 1.8
-The next instructions were taken from [golang site](https://golang.org/doc/install)
+In order to install Clear Containers on RHEL 7.3, just run:
 
 ```
-$ curl -L -O https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz
-$ sudo tar -C /usr/local -xzf go1.8.linux-amd64.tar.gz
-$ export PATH=$PATH:/usr/local/go/bin
-$ mkdir ~/go
-$ export GOPATH=~/go
+$ ./installation/rhel-setup.sh
 
 ```
+Note:
+This script will install all the packages required by cc-oci-runtime.
 
-## Install the clear containers runtime from sources
+## Verify
 
-```
-$ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
-$ export PATH=/usr/local/gcc-6.2.0/bin:$PATH
-$ sudo yum install libuuid-devel pcre-devel libffi-devel zlib-devel gettext-devel pkgconfig libmnl-dev libmnl
-$ go get github.com/01org/cc-oci-runtime
-$ pushd go/src/github.com/01org/cc-oci-runtime
-$ git checkout tags/2.1.0
-$ ./autogen.sh \
-        --prefix=/usr/local \
-        --disable-tests \
-        --disable-functional-tests \
-        --with-cc-image=/usr/share/clear-containers/clear-containers.img \
-        --with-cc-kernel=/usr/share/clear-containers/vmlinux.container \
-        --with-qemu-path=/usr/local/bin/qemu-system-x86_64
-$ make -j5 && sudo make install
-$ popd
+### Check cc-oci-runtime version
 
 ```
-
-## Docker
-
-Currently cc-oci-runtime supports docker 1.12.1 to 1.12.6.
-
-For more information on docker installation, see:
-- https://docs.docker.com/engine/installation/linux/rhel/
-
-
-## Configure docker to use Clear Containers by default
+$ cc-oci-runtime --version
 
 ```
-$ sudo mkdir -p /etc/systemd/system/docker.service.d/
-$ sudo vi /etc/systemd/system/docker.service.d/clr-containers.conf
-[Service]
-ExecStart=
-ExecStart=/usr/bin/dockerd -D --add-runtime cor=/usr/local/bin/cc-oci-runtime --default-runtime=cor
-```
 
-## Restart the docker systemd service
+### Restart the docker systemd service
 
 ```
 $ sudo systemctl daemon-reload
 $ sudo systemctl restart docker
+
 ```
 
-## Start cc-proxy socket
+### Start cc-proxy socket
 
 ```
 $ sudo systemctl start cc-proxy.socket
+
 ```
 
-## Run a Clear Container
-You are now ready to run clear containers. For example:
+### Run a Clear Container
```

Comment:
Change lines 45 and 46 to:
4. Run an example Clear container with the following command: